src/icons/README.md

```diff
@@ -1,12 +1,15 @@
 # qBittorrent Icons
 
-Initial `qbt-theme` icons created by Bert Verhelst (<verhelstbert@gmail.com>).
-
-Icons are based on the `Font-Awesome` icon-set: [link](http://fontawesome.io/icons/).
-
-If you need to add an icon that qBittorrent does not already use, you can take an icon from the SVG fork of `Font-Awesome`: [link](https://github.com/encharm/Font-Awesome-SVG-PNG).
-
-
-## Optimizing SVG
-
-Use [svgcleaner](https://github.com/RazrFalcon/svgcleaner)
+## Previous
+Old icons were created by Bert Verhelst (<verhelstbert@gmail.com>).
+Icons were based on the [Font-Awesome](https://fontawesome.com/icons) icon set.
+
+## Redesign
+Redesigned icons has been created by Nowshed H. Imran (<nowshed.imran@gmail.com>). There are some additional icons available in [flow-icon](https://github.com/nowshed-imran/flow-icon) repository that can be used in future development.
+New icons are based on [la-capitaine-icon-theme](https://github.com/keeferrourke/la-capitaine-icon-theme) and [ionicons](https://github.com/ionic-team/ionicons).
+`la-capitaine` project is owned by Keefer Rourke and due permission was received before modifying them.
+`ionicons` project is under MIT license and thus it can be used freely.
+Both of these projects can be used as reference for future icon designs.
+
+## Color Reference
+Colors have been selected from QT documented [SVG Colors](https://doc.qt.io/qt-6/qcolorconstants.html#svg-colors). It is recommended to use color Hex from the given source.
```

Comment:
@xavier2k6 Should "QT" be written as "Qt" or not?

Reply:
Yes, it should be Qt